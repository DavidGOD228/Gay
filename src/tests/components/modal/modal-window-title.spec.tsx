import 'react-native'
// Note: test renderer must be required after react-native.
import React from 'react'
import renderer from 'react-test-renderer'
import ModalWindowTitle from '../../../components/modal/modal-window-title'
import {
  Text
} from 'react-native'

it('renders correctly empty', () => {
  const tree = renderer.create(
    <ModalWindowTitle />
  )
  expect(tree)
    .toMatchSnapshot()
})

it('renders correctly text', () => {
  const tree = renderer.create(
    <ModalWindowTitle >
      <Text>
        All you had to do was follow the damn train CJ
      </Text>
    </ModalWindowTitle>
  )
  expect(tree)
    .toMatchSnapshot()
})
