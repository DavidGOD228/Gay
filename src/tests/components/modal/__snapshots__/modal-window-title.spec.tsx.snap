// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly empty 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  disabled={false}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "white",
      "fontSize": 24,
      "fontWeight": "bold",
    }
  }
/>
`;

exports[`renders correctly text 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  disabled={false}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "white",
      "fontSize": 24,
      "fontWeight": "bold",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    All you had to do was follow the damn train CJ
  </Text>
</Text>
`;
